sos/report/plugins/flatpak.py

```diff
@@ -20,19 +20,27 @@ class Flatpak(Plugin, IndependentPlugin):
 
     def setup(self):
         env = {"GVFS_REMOTE_VOLUME_MONITOR_IGNORE": "1"}
+        gvfs_pred = SoSPredicate(
+            self, cmd_outputs={
+                'cmd': 'systemctl --user status gvfs-daemon.service',
+                'output': '(running)'
+            }
```

Comment:
`SosPredicate()` supports a ` service` parameter to check for running services. Is that not sufficient here?

Reply:
I was thinking about it, but I think the `--user` option is crucial difference here. Let me re-test it..

Reply:
As we get services list via `systemctl list-unit-files --type=service` and that output does not contain users' services, `InitSystem` is not aware of the service despite it is running. So `service` parameter is useless here /o\.

Reply:
Ah, ok, good observation. I guess that makes the question, should we expand our profiling for `SystemdInit` to include user services, or just go forward with this as written and keep our SystemD abstraction in the dark about user services?